cfme/tests/infrastructure/test_instance_analysis.py

```diff
@@ -452,3 +494,33 @@ def test_ssa_packages(provider, instance, soft_assert):
         if packages.find_row('Name', package_name):
             return
     pytest.fail("Package {0} was not found".format(package_name))
+
+
+@pytest.mark.long_running
+def test_ssa_files(provider, instance, policy_profile, soft_assert):
+    """Tests that instances can be scanned for specific file."""
+
+    if instance.system_type == WINDOWS:
+        pytest.skip("We cannot verify Windows files yet")
+
+    instance.smartstate_scan()
+    wait_for(lambda: is_vm_analysis_finished(instance.name),
+             delay=15, timeout="10m", fail_func=lambda: tb.select('Reload'))
+
+    # Check that all data has been fetched
+    current = instance.get_detail(properties=('Configuration', 'Files'))
+    assert current != '0', "No files were scanned"
+
+    sel.click(InfoBlock("Configuration", "Files"))
```

Comment:
We should try to turn these into navigatable steps rather than invoking sel here......right now I think it's probably OK, but in the future I'd like to avoid calling raw clicks in test. @mfalesni thoughts?